pyuploadcare/client.py

```diff
@@ -605,8 +604,23 @@ def _extract_uuids(
             file_ if isinstance(file_, File) else File(file_, self)
             for file_ in files
         )
-        uuids: List[str] = extracts_uuids(file_generator)
 
+        def __extracts_uuids(files: Iterable[Union[str, File]]) -> List[str]:
```

Comment:
moved here to avoid circular dependencies.
it's not being used anywhere else.

maybe rewrite without inner function